beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -187,7 +193,10 @@ pub enum BlockError<T: EthSpec> {
     ///
     /// The `proposer` has already proposed a block at this slot. The existing block may or may not
     /// be equal to the given block.
-    RepeatProposal { proposer: u64, slot: Slot },
+    RepeatProposal {
```

Comment:
Including this definition, there are 5 references to `RepeatProposal` which need removing/updating. The ones in the tests should be adjusted to `SlashableProposal`. We could add the slot/proposer to that error variant to retain the same level of detail.